Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -70,6 +70,15 @@ public override void Initialize()
 
         private void OnPendingMapInit(EntityUid uid, PendingZombieComponent component, MapInitEvent args)
         {
+            if (!TryComp<MobStateComponent>(uid, out var mobState))
+                return;
+
+            if (_mobState.IsDead(uid, mobState) && !HasComp<ZombieComponent>(uid) && !HasComp<ZombieImmuneComponent>(uid))
+            {
+                ZombifyEntity(uid);
```

Comment:
i'm pretty sure zombify ent checks for zombie comp and immune comp and if it doesn't it probably should?

Reply:
yeah